tests/test_crop_production.py

```diff
@@ -333,4 +334,5 @@ def test_missing_keys_regression(self):
         expected_missing_keys = set(
             self.base_required_keys +
             ['fertilization_rate_table_path'])
-        self.assertEqual(invalid_keys, expected_missing_keys)
\ No newline at end of file
+        self.assertEqual(invalid_keys, expected_missing_keys)
+

```

Comment:
Hey sorry @dcdenu4 I think maybe this is at the end of the file?  If so, doesn't PEP8 call for no blank lines at the end?  Sorry again for the triviality of this!

Reply:
Great question, I'll check. Pycodestyle was sometimes suggesting a line at the end and sometimes not... I'll check PEP8